views/js/portableLib/OAT/util/html.js

```diff
@@ -24,12 +24,6 @@ define([
 
     return {
         render : function render($container){
-            var markup = $container.html();
-
-            //remove xml ns (would break jquery selector otherwise)
-            markup = xml.removeNamespace(markup);
```

Comment:
If you don't use `xml` anymore, could you remove 'taoQtiItem/portableLib/OAT/util/xml', please?